edgelet/edgelet-core/src/module.rs

```diff
@@ -139,15 +139,15 @@ impl ModuleRuntimeState {
 
 #[derive(serde_derive::Deserialize, Debug, serde_derive::Serialize)]
 pub struct ModuleSpec<T> {
-    name: String,
+    pub name: String,
```

Comment:
(general comment, not necessarily part of this PR)

If we are making all these struct fields `pub` (IMO, as they should have been originally), then we aught to remove the getter/setter interface entirely. This shouldn't be difficult, though it will be a bit tedious... I guess we can do it as part of the Tokio 0.1 to 0.3/1.0 rewrite effort?

Reply:
Yes. Removing the getters will mean a lot more ancillary changes. Otherwise I would've removed them; I'm not a fan of trivial getters either.

Reply:
I am fine with both for now. I added trivial getters for my PR because I knew they will be rolled into structs eventually.